packages/venia-concept/webpack.config.js

```diff
@@ -114,13 +116,32 @@ module.exports = async env => {
     const browserPlugins = new Set()
         .add('HtmlWebpackPlugin')
         .add('LocalizationPlugin')
-        .add('ServiceWorkerPlugin');
+        .add('ServiceWorkerPlugin')
+        .add('VirtualModulesPlugin')
+        .add('WebpackAssetsManifest');
 
     // remove browser-only plugins
     serverConfig.plugins = serverConfig.plugins.filter(
         plugin => !browserPlugins.has(plugin.constructor.name)
     );
 
+    // remove browser-only module rules
+    serverConfig.module.rules = serverConfig.module.rules.map(rule => {
+        if (rule.oneOf) {
```

Comment:
I'm okay with this, but it's a bit fragile. Could we look for the module rule we want to remove by its `test` value rather than its `oneOf` value? Would that work?